src/misc.ts

```diff
@@ -353,6 +354,16 @@ export function findInEnum<TEnum extends {}>(needle: string, haystack: TEnum) {
   throw new Error(`Key ${needle} not found in enum`);
 }
 
+export function addressToAddressInput<T>(
+  address: T & OrderDetails_order_shippingAddress
+): AddressInput {
+  const { id, __typename, ...rest } = address;
+  return {
+    ...rest,
+    country: findInEnum(address?.country?.code, CountryCode)
+  };
+}
+
```

Comment:
So now in `misc` there are 3 functions which somehow transform address to address input (`transformFormToAddressInput`, `transformAddressToAddressInput`, `addressToAddressInput`). What are the differences between them, is it possible to merge them?

Reply:
`transformFormToAddressInput` returns `T & AddressInput`
`transformAddressToAddressInput` returns `AddressInput` and merges customer data
`addressToAddressInput` returns `AddressInput` without `id` and `__typename`

I would rather keep them separate